src/main/java/data_access/StudentFirestoreUserDataStudentAccessObject.java

```diff
@@ -33,17 +35,17 @@
  * This implementation uses Firebase and only persists data regarding the
  * Student entity
  */
-public class StudentFirestoreUserDataAccessObject implements StudentLoginDataAccessInterface,
+public class StudentFirestoreUserDataStudentAccessObject implements StudentLoginDataAccessInterface,
```

Comment:
Was there a reason to renaming the DAO to have student, when it's already in the front?

Reply:
I think it was the refactoring playing tricks on me. Will fix